roles/jd-client/README.md

```diff
@@ -10,7 +10,7 @@ downstream, so that everything that is dowstream do not need to know that job is
 
 ## Setup
 ### Configuration File
-The `proxy-config-example.toml` is a configuration example.
+The `jdc-config-local-example.toml` is a configuration example.
```

Comment:
I would say that there are 2 config files in config-examples/ dir:
1) jdc-config-local-example.toml
2) jdc-config-hosted-example.toml
The difference between them is that the second one uses our community-hosted jd-server (JDS)